airbyte-cdk/python/airbyte_cdk/sources/streams/http/exceptions.py

```diff
@@ -9,7 +9,9 @@
 
 
 class BaseBackoffException(requests.exceptions.HTTPError):
-    pass
+    def __init__(self, request: requests.PreparedRequest, response: requests.Response):
+        error_message = f"Request URL: {request.url}, Response Code: {response.status_code}, Response Text: {response.text}"
+        super().__init__(error_message)
```

Comment:
can you pass `request=request, response=response` as named parameters here as well?

Reply:
https://github.com/airbytehq/airbyte/pull/11653